libs/blocks/graybox/graybox.js

```diff
@@ -0,0 +1,253 @@
+import { createTag, getMetadata, MILO_EVENTS } from '../../utils/utils.js';
+import { getModal, closeModal } from '../modal/modal.js';
+import { iphoneFrame, ipadFrame } from './mobileFrames.js';
+
+const OPTION = {
+  CHANGED: 'changed',
+  NO_CLICK: 'no-click',
+};
+
+const CLASS = {
+  CHANGED: 'gb-changed',
+  GRAYBOX_BODY: 'gb-graybox-body',
+  NO_BORDER: 'gb-no-border',
+  NO_CHANGE: 'gb-no-change',
+  NO_CLICK: 'gb-no-click',
+  NO_OVERLAY: 'gb-no-overlay',
+  OVERLAY: 'gb-overlay',
+  PHONE_PREVIEW: 'gb-phone-preview',
+  TABLET_PREVIEW: 'gb-tablet-preview',
+};
+
+const METADATA = {
+  DESC: 'gb-desc',
+  FOOTER: 'gb-footer',
+  GNAV: 'gb-gnav',
+  TITLE: 'gb-title',
+};
+
+const USER_AGENT = {
+  iPhone: 'Mozilla/5.0 (iPhone13,2; U; CPU iPhone OS 14_0 like Mac OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) Version/10.0 Mobile/15E148 Safari/602.1',
+  iPad: 'Mozilla/5.0 (iPad; CPU OS 13_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.3 Mobile/15E148 Safari/604.1',
+};
+
+let deviceModal;
+
+const isMobileDevice = () => /Android|webOS|iPhone|iPad|iPod/i.test(navigator.userAgent);
+
+const getTableValues = (el) => [...el.childNodes].reduce((rdx, row) => {
+  if (row.children) {
```

Comment:
I'm a fan of early returns in favour of if/else nesting
```suggestion
  if (!row.children) return rdx
```